ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,199 @@
+            setupSubtitleLabel(using: subtitle)
+        } else if let identifier = identifier {
+            setupAvatarView(with: identifier)
+            setupTitleLabel(using: identifier)
+        }
+
+        backgroundColor = Colors.surfacePrimary
+        setupLayout()
+    }
+
+    public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private func setupAvatarView(with firstName: String, and lastName: String) {
+        let fullName = firstName + " " + lastName
+        avatarView.setup(primaryText: fullName, secondaryText: nil, image: avatarImage)
+    }
+
+    private func setupAvatarView(with identifier: String) {
+        avatarView.setup(primaryText: identifier, secondaryText: nil, image: avatarImage)
+    }
+
+    private func setupLayout() {
+        var constraints = [NSLayoutConstraint]()
+        constraints.append(widthAnchor.constraint(equalToConstant: Constants.contactWidth))
+        constraints.append(contentsOf: avatarLayoutConstraints())
+
+        avatarView.translatesAutoresizingMaskIntoConstraints = false
+        labelContainer.addSubview(avatarView)
+
+        if let subtitleLabel = subtitleLabel {
+            constraints.append(contentsOf: titleLabelLayoutConstraints())
+            constraints.append(contentsOf: subtitleLabelLayoutConstraints())
+            subtitleLabel.translatesAutoresizingMaskIntoConstraints = false
+            labelContainer.addSubview(subtitleLabel)
+        } else {
+            constraints.append(contentsOf: identifierLayoutConstraints())
+        }
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        labelContainer.addSubview(titleLabel)
+        
+        labelContainer.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(labelContainer)
+
+        constraints.append(contentsOf: labelContainerLayoutConstraints())
+        NSLayoutConstraint.activate(constraints)
+    }
+
+    private func avatarLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            avatarView.heightAnchor.constraint(equalToConstant: Constants.avatarViewHeight),
+            avatarView.widthAnchor.constraint(equalToConstant: Constants.avatarViewWidth),
+            avatarView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            avatarView.topAnchor.constraint(equalTo: topAnchor),
+            avatarView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            avatarView.trailingAnchor.constraint(equalTo: trailingAnchor)
+        ]
+    }
+
+    private func labelContainerLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            labelContainer.widthAnchor.constraint(equalToConstant: Constants.contactWidth),
+            labelContainer.centerXAnchor.constraint(equalTo: centerXAnchor),
+            labelContainer.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndFirstLabel),
```

Comment:
avatarView is a subview of labelContainer, this constraint doesn't really make much sense.
Is it possible that avatarView was supposed to be a subview of self instead?

Reply:
Yes, you're correct. `labelContainer` should have only contained the `titleLabel` and `subtitleLabel`. `avatarView` is supposed to be a subview of `self`. I have that fixed now in a commit.